### PR TITLE
chore: update component tests structure for snapchat custom audience

### PR DESCRIPTION
**Pull request by aashishmalik**
## What are the changes introduced in this PR?

update component tests structure for snapchat custom audience

## What is the related Linear task?

Resolves INT-1494

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3125/test-report.html

**Comment by utsabc** [comment]
Lets also add `other.ts` test check `braze` for reference

File: test/integrations/destinations/snapchat_custom_audience/dataDelivery/oauth.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+            ],
+            statTags: {
+              errorCategory: 'network',
+              errorType: 'aborted',
+              destType: 'SNAPCHAT_CUSTOM_AUDIENCE',
+              module: 'destination',
+              implementation: 'native',
+              feature: 'dataDelivery',
+              destinationId: 'default-destinationId',
+              workspaceId: 'default-workspaceId',
```
[the remark this answers is not in the document]
Comment (aashishmalik) [reply test/integrations/destinations/snapchat_custom_audience/dataDelivery/oauth.ts:266]: decalred constants

File: test/integrations/destinations/snapchat_custom_audience/dataDelivery/data.ts [hunk 2]
```diff
@@ -1,206 +1,4 @@
-export const data = [
```
[the remark this answers is not in the document]
Comment (aashishmalik) [reply test/integrations/destinations/snapchat_custom_audience/dataDelivery/data.ts:1]: not removed this ... bundled them in v0 test cases to diff files

File: test/integrations/destinations/snapchat_custom_audience/dataDelivery/business.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      },
+    ],
+  },
+};
+
+export const businessTestScenarios = [
+  {
+    id: 'snapchat_custom_audience_v0_oauth_scenario_1',
+    name: 'snapchat_custom_audience',
+    description: '[Proxy v0 API] :: successfull call',
```
[the remark this answers is not in the document]
Comment (aashishmalik) [reply test/integrations/destinations/snapchat_custom_audience/dataDelivery/business.ts:30]: no batchinh supported for this destination

**Review by utsabc — APPROVED** [review]

**Review by mihir-4116 — APPROVED** [review]

**Pull request merged**